src/uas/UAS.cc

```diff
@@ -2827,21 +2825,6 @@ void UAS::enableHilXPlane(bool enable)
         }
         qDebug() << "CREATED NEW XPLANE LINK";
         simulation = new QGCXPlaneLink(this);
-
-        float noise_scaler = 0.1f;

```

Comment:
I think we're over-optimising here. We neither need completely realistic flight dynamics nor a choice of different aircraft. We need a single model whose flight dynamics, including noise, get "close" for the common flight situations *and* which is easy enough to use so average users are able to benefit from it.
Right now we have a single config (the Rascal) which works, and we need to tune the noise distribution so that it matches the in-flight situation as close as possible without throwing the estimator too much off because it runs slower.

We have to be realistic. We neither have the resources to maintain N configs nor do we have the resources to get the noise distribution perfect. By just getting the Rascal "close" we would however do substantially better than what we have now.